### ISSUE TITLE
Job Specific Storage Requirements

### ISSUE THREAD

**Issue by johnbradley** [Duke-GCB/lando#43]
We expect jobs to have variable and typically really large disk requirements.
To address this we could create cinder volumes for each job VM.
It is trivial to create a cinder volume using the shade library, but the volume will need to be formatted.
Example commands to format and mount storage at `/mnt/storage`:
```
#find the name of our disk
ls /dev/disk/by-id/virtio-*
sudo mkfs.ext4 /dev/disk/by-id/virtio-...
sudo mount /dev/disk/by-id/virtio-... /mnt/storage
```
The name of the disk seems to be based on the beginning of the volume id.

**Comment by johnbradley** [comment]
The shade `attach_volume` method returns an object that has a key 'device' that contains the traditional device label. Formatting this volume would be possible in the VM boot script.

**Comment by dleehr** [comment]
#44 allows the root disk of a VM to be a volume of arbitrary size, so it need not be separately attached/formatted. Is this issue still relevant?

**Comment by johnbradley** [comment]
This has been resolved by #44
